mover.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"github.com/k0kubun/go-ansi"
+	"github.com/schollz/progressbar/v3"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+func modifyFileName(targetDir string, imageInfo ImageInfo) string {
+	fileName := imageInfo.Info.Name()
+	extension := filepath.Ext(fileName)
+	name := fileName[:len(fileName)-len(extension)]
+
+	// Create the target path
+	targetPath := targetDir + "/" + name + strings.ToLower(extension)
```

Comment:
I'd replace all of these lines with `filepath.Join`:
```suggestion
	targetPath := filepath.Join(targetDir, name + strings.ToLower(extension))
```